frontend/src/metabase/embedding-sdk/types/components/data-picker.ts

```diff
@@ -0,0 +1,18 @@
+import type Database from "metabase-lib/v1/metadata/Database";
+import type { TableId } from "metabase-types/api";
+
+export interface DataSourceSelectorProps {
```

Comment:
The interface is based on the props we pass to it.

https://github.com/metabase/metabase/blob/7f790fe68ca323340b207d4cbbdfa2e3a81ce0cc/frontend/src/metabase/querying/notebook/components/NotebookDataPicker/EmbeddingDataPicker/EmbeddingDataPicker.tsx#L100-L121